### DISCUSSION TITLE
Does win7 support oh-my-posh

### DISCUSSION THREAD

**Discussion by llplay** [JanDeDobbeleer/oh-my-posh#1094] [Troubleshoot]
I use win7,  install followed by the site: https://ohmyposh.dev/docs/pwsh,  but it does not work correctly,  what should i do?
![image](https://user-images.githubusercontent.com/19726656/138084922-e4902414-a88c-44f2-a0a6-f0cbfd7ffdfb.png)

my powershell version:
PSVersion 5.1.14409.1005 
PSEdition Desktop PSCompatibleVersions {1.0, 2.0, 3.0, 4.0...} 
BuildVersion 10.0.14409.1005 
CLRVersion 4.0.30319.42000 
WSManStackVersion 3.0 
PSRemotingProtocolVersion 2.3 
SerializationVersion 1.1.0.1

**Comment by JanDeDobbeleer** [comment]
Interesting, it's not Windows 7 per se but the terminal that doesn't understand ANSI (which was indeed an issue before). I'm not actively supporting it, as this is supposed to be treated as deprecated and insecure, but you can have a look for another terminal such as ConEmu which supports ANSI rendering.

**Comment by chenyulue** [comment]
The similar problem in win7:

terminal: tabby
powershell: 7.2.10
terminal font:  FiraCode Nerd Font Mono

It seems the terminal renders the prompt uncorrectly, without colors rendered. I also tried ConEmu, which had the similar problem.

Tabby and powershell 7.2.10 work well on win10. 

Do you have any ideas? Thanks! @JanDeDobbeleer 

![微信图片_20230313142316](https://user-images.githubusercontent.com/44137770/224623364-c286d69c-acda-43b1-9f52-031ca3dd6dd2.jpg)

**Reply by JanDeDobbeleer** [reply]
@chenyulue Win7's conhost doesn't support ANSI. So this isn't supported AND win7 is deprecated since January 14, 2020.
